data/json/construction.json

```diff
@@ -2349,5 +2349,16 @@
     ],
     "pre_terrain": "t_fence_post",
     "post_terrain": "t_wall_rammed_earth"
+  },
+  {
+    "type": "construction",
+    "description": "Hang Hanging Meathook",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 2 ] ],
+    "time": 30,
```

Comment:
30 minutes and fabrication 2 just to hang a chain and attach hook to it?

Reply:
My assumption was that it would require welding to attach a hook to it, and that welding is non-trivial.

Reply:
As it stands you're doing it with a hammer and bolts according to the tools you've required, which frankly makes more sense than welding anyway.

Reply:
I have made it a fab 1 and 15 minutes to hang the meathook, and so will resolve this convo